astronomy/horizons.awk

```diff
@@ -0,0 +1,50 @@
+BEGIN {
```

Comment:
Please put comments explaining what this script does, what input it expects, and what output it produces.  The next maintainer will thank you.